.storybook/main.ts

```diff
@@ -2,38 +2,31 @@ import type { StorybookConfig } from "@storybook/react-webpack5";
 import { storybookAddonStylingWebpackConfig } from "./addon-styling-webpack-config";
 
 const config: StorybookConfig = {
-  features: {
-    storyStoreV7: true,
-  },
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
   staticDirs: ["../static"],
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/addon-a11y",
     "@storybook/addon-interactions",
-    "storybook-addon-performance",
```

Comment:
I'm guessing this doesn't have a Storybook v8 compatible version yet and this is the reason it is removed, is that correct?

Reply:
Yes, that's correct, it only works for Storybook < 8 apparently.